### PR TITLE
Updated per conversation with Richard

### LINKED ISSUE #1598: Insecure registry documentation needs updates
Some bad mechanics in the sentence structures --- not starting with caps that kind of thing.

Check on this.
https://twitter.com/nsturnitin/status/717431474257956865

### PR DESCRIPTION
**Pull request by moxiegirl**
Fixes #1598

Signed-off-by: Mary Anthony <mary@docker.com>

@RichardScothern  for review.  The error was coming from the .tmpl file.  Links to the `manifest*` files didn't resolve because they are listed as draft.  Fixed this too.

### CODE REVIEW

File: docs/insecure.md [hunk 1]
```diff
@@ -8,83 +8,104 @@ keywords = ["registry, on-prem, images, tags, repository, distribution, insecure
...
+You have to understand the downsides in doing so, and the extra burden in
+configuration.
 
 ## Deploying a plain HTTP registry
 
-> **Warning**: it's not possible to use an insecure registry with basic authentication
+> **Warning**: it's not possible to use an insecure registry with basic authentication.
 
 This basically tells Docker to entirely disregard security for your registry.
+While this it is relatively easy to configure the daemon in this way, it is
```
Comment (RichardScothern) [inline docs/insecure.md]: extraneous 'this'

File: docs/insecure.md [hunk 2]
```diff
@@ -8,83 +8,104 @@ keywords = ["registry, on-prem, images, tags, repository, distribution, insecure
...
- 
- - this is **very** insecure: you are basically exposing yourself to trivial MITM, and this solution should only be used for isolated testing or in a tightly controlled, air-gapped environment
- - you have to configure every docker daemon that wants to access your registry 
-  
 ## Using self-signed certificates
 
 > **Warning**: using this along with basic authentication requires to **also** trust the certificate into the OS cert store for some versions of docker (see below)
 
-Generate your own certificate:
+This is more secure than the insecure registry solution.  You must you have to configure every docker daemon that wants to access your registry
```
Comment (RichardScothern) [inline docs/insecure.md]: extraneous 'you have to'

**Comment by RichardScothern** [comment]
A couple of minor points, but overall LGTM

**Comment by moxiegirl** [comment]
@RichardScothern ok, your comments are in and I've also done another visual pass and tweaked a couple of format things I found.

**Pull request merged**